src/FirebaseProject.php

```diff
@@ -0,0 +1,106 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Kreait\Laravel\Firebase;
+
+use Kreait\Firebase;
+
+class FirebaseProject
+{
+    /** @var \Kreait\Firebase\Factory $factory */
```

Comment:
I usually don't include FQNs in PHPDocs, so this "risks" being changed to `Kreait\Firebase\Factory` (due to the existing import) or to `Factory` with an additional `use` statement somewhen in the future 🙈  (similarly elsewhere)